python/whylogs/core/schema.py

```diff
@@ -164,6 +173,13 @@ class ColumnSchema:
     cfg: MetricConfig = MetricConfig()
     type_mapper: TypeMapper = StandardTypeMapper()
     resolver: Resolver = StandardResolver()
+    validators: Dict[str, Validator] = field(default_factory=dict)
 
     def get_metrics(self, name: str) -> Dict[str, Metric]:
         return self.resolver.resolve(name=name, why_type=self.type_mapper(self.dtype), column_schema=self)
+
+    def get_validators(self, name: str) -> Set[Validator]:
```

Comment:
yeah that was wrong, but it's actually a list of Validators, so I figured it would be best to stick with get_validators() in the plural, and change the return to be `List[Optional[Validator]]`.